source/auth/auth.rst

```diff
@@ -1210,24 +1210,12 @@ username.
     description: "Server's reply to clientStep1"
     strict: false
     fields:
-      authorizationEndpoint:
+      issuer:
         description: >-
-          URL where the IDP may be contacted for end user
-          authentication and authorization code generation.
-        type: string
-        optional: true # Req if deviceAuthorizeEndpoint not present
-      tokenEndpoint:
-        description: >-
-          URL where the IDP may be contacted for authorization
-          code <=> ID/access token exchange.
-        type: string
-        optional: true # Req if deviceAuthorizeEndpoint not present
-      deviceAuthorizationEndpoint:
-        description: >-
-          URL where the IDP may be contacted for device
-          authentication and authorization code generation.
-        type: string
-        optional: true # Req if authorizeEndpoint not present
```

Comment:
`clientSecret` below also needs to be dropped to match [SERVER-75122](https://jira.mongodb.org/browse/SERVER-75122), right?

Reply:
Good call out, thanks! Fixed